backend/akr/src/test/java/fi/oph/akr/scheduled/ExpiringAuthorisationsEmailCreatorTest.java

```diff
@@ -105,12 +113,19 @@ private Authorisation createAuthorisation(
     final LocalDate termEndDate,
     final String translatorEmail
   ) {
-    final Translator translator = Factory.translator();
+    final Translator translator = createTranslator(translatorEmail);
+    return createAuthorisation(translator, meetingDate, termEndDate);
+  }
+
+  private Authorisation createAuthorisation(
```

Comment:
Voisi ehkä nimetä `createAuthorisationForTranslator`

Reply:
Ihan hyvä näin, parametrit erottaa riittävästi.